### ISSUE TITLE
autoValue not recognizing Session.get()

### ISSUE THREAD

**Issue by nickeleres** [aldeed/meteor-autoform#677]
Spoke to a couple people who recommended I bring this here as a bug.  I am setting a session variable here in the `onBeforeAction` method

         Router.route('singleDetails', {
			path: '/speakers/singleDetails/:_id',
			template: 'singleDetails',
			data: function(){
				return Speakers.findOne({_id: this.params._id});
			},
			onBeforeAction: function(){
				console.log(this.params._id);
				Session.set('pathId', this.params._id);
				this.next();
			}
		});

		Template.singleDetails.helpers({

			singleDetails: function(){
				return singleDetails.find({});
			}
		});

But when I try to `get` it in my schema, it console.log's to null and it gives me a server error

        speakerId: {
				type: String,
				autoValue: function(){
					Session.get('idPath'):
				}
			}

In short: I am not able to dynamically set a schema field to `this.params._id`

**Comment by jakec729** [comment]
Was a solution reached for this?

**Comment by nickeleres** [comment]
Yeah it was. I ended up pursuing a different approach. Thanks for following
up,

Nick
